rtm.go

```diff
@@ -12,7 +12,10 @@ import (
 //
 // To have a fully managed Websocket connection, use `NewRTM`, and call `ManageConnection()` on it.
 func (api *Client) StartRTM() (info *Info, websocketURL string, err error) {
-	return api.StartRTMContext(context.Background())
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
Minor - can we move the timeout to a const?

Reply:
Sure, done